add volume konwn issue docs (#366)
harvester/docs: versioned_docs/version-v1.1/volume/create-volume.md

1. Select an existing `Image`.
1. Configure the `Size` of the volume.

![create-image-volume](/img/v1.1/volume/create-image-volume.png)

## Known Issues

> ### Volume created is not appearing in the volume list

Reviewer: ```suggestion
### The Volumes page does not show the created volume
```

> On the volume list page, users with 'project-member' permissions cannot see the created volumes.

Reviewer: ```suggestion
**Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.
**Workaround**: You can refresh the **Volumes** page and the volume should display as expected.
```

Reviewer: 1. Consider adding the affected versions
2. Not sure if refreshing the page or applying the `harvester-v1.1.2-patch1` is the correct workaround. If applying the patch is the workaround, please add the instructions as well
3. Consider linking to the github issue page

1. Select an existing `Image`.
1. Configure the `Size` of the volume.

![create-image-volume](/img/v1.1/volume/create-image-volume.png)

## Known Issues

| Summary                                                                                                                                                                                   | Status    | Last Updated |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
| [The Volumes page does not show the created volume](https://github.com/harvester/harvester/issues/3874) | Resolved | 2023-07-28   |

### The Volumes page does not show the created volume

> **Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.

Reviewer 2: ```suggestion
**Summary**: After creating a volume when using Harvester from Rancher, users with the project role **Project Member** cannot find the newly created volume on the **Volumes** page.

**Affected versions**:
- Harvester: v1.1.2

| Status    | Last updated |
|-----------|--------------|

> | Resolved(Harvester > v1.1.2)  | 2023-07-28   |

Reviewer: ```suggestion
| Resolved (Harvester > v1.1.2)  | 2023-07-28   |
```

**Workaround**: You can temporarily change the Harvester plugin version to [v1.2.1-pitch1](https://github.com/harvester/dashboard/releases/tag/v1.1.2-patch1) from the Harvester UI.

> 1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.

Author: Go to to ?      @LucasSaintarbor

> 1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.

Reviewer 2: @WuJun2016 Fixed. Thanks for catching that

> 2. Change the `value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`

Reviewer: ```suggestion
2. Change `Value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`.
```

3. In the `Settings` list, find `ui-source` and then click `⋮`  > `Edit Setting`.

> 4. Change `value` to `External`.

Reviewer 2: ```suggestion
4. Change the **Value** to **External** to use an external UI source.
```

> 4. Change the `value` to `External`.

Reviewer: ```suggestion
4. Change `Value` to `External`.
```

> 5. Log in again using the **project_member** user in Rancher.

Reviewer 2: ```suggestion
5. Log in again as a **Project Member** user in Rancher to view the newly created volume for your Harvester cluster. 
```
